server/routes.js

```diff
@@ -298,11 +316,13 @@ exports.isomorphicRoutes = function isomorphicRoutes(
     prerenderServiceUrl = "",
     redirectToLowercaseSlugs = false,
     shouldEncodeAmpUri,
-    sMaxAge = "900",
+    sMaxAge = 900,
```

Comment:
Don't think changing this to number is a good idea because of 2 reasons
- it's ultimately used as string [here](https://github.com/quintype/quintype-node-framework/blob/1d0cd8b31ee72575437e07bd01e833640f6cf8cf/server/handlers/cdn-caching.js#L39)
- Existing publishers are already passing it as a string. They'll have to make this change again [Example](https://github.com/quintype/newslaundry/blob/a5593b1360f6aa7f8e0249668c7afc9278d0692f/app/server/app.js#L142)